### DISCUSSION TITLE
AppImage with Planify

### DISCUSSION THREAD

**Discussion by strogiyotec** [alainm23/planify#1626] [General]
**Is your feature request related to a problem? Please describe.**
I am using Debian and I don't have Vala and Flatpack installed. As planner has a lot of dependencies It would be cool to have an AppImage with fully featured Planner. Thanks in advance 
**Describe the solution you'd like**
AppImage file with Panner 

**Describe alternatives you've considered**
UPload Planner into apt store

**Comment by Abhinav1217** [comment]
Is this being considered? 

Flatpak needs to install over 1 GB of platform for a 27MB of planner app.
![Screenshot from 2021-12-01 16-09-09](https://user-images.githubusercontent.com/740584/144220218-9473d333-566d-47ba-872f-fc00c3ea9c35.png)

Even if Appimage bundles some of the Pantheon dependencies with it, it should be way smaller than flatpak. Not to mention the overhead of cold boot for flatpak or snaps.

**Comment by shuvashish76** [comment]
Coming from https://github.com/edfloreshz/done/issues/10,  which is archived today & [recommends](https://github.com/edfloreshz/done#this-repository-is-no-longer-maintained-please-switch-over-to-planify) this project.
+1 for AppImage

**Comment by directentis1** [comment]
I also feel the same, AppImages is much lighter and more portable than Flatpak or Snap which are too heavy when I use it in  Linux through Kasm Web, which is light in size (because they run through Docker).

**Comment by wqk151** [comment]
The runtime of flatpak is too large
+1 for AppImage

**Reply by byquanton** [reply]
Planify uses many parts of the runtime, so all of those libraries would also need to be present in the AppImage in order to create it. I don't think an AppImage would be much smaller.

**Reply by Samueru-sama** [reply]
> Planify uses many parts of the runtime, so all of those libraries would also need to be present in the AppImage in order to create it. I don't think an AppImage would be much smaller.

The invidual app will be smaller as a flatpak (usually), the problem is the runtimes: https://pkgforge-dev.github.io/Anylinux-AppImages/disk-usage-vs-flatpak.html

**Comment by alainm23** [comment]
Thanks for the suggestion! Flatpak is the recommended way to install Planify on Debian and other distributions — it bundles all dependencies so you don't need Vala or any other build tools.

```
sudo apt install flatpak
flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
flatpak install flathub io.github.alainm23.planify
```

We don't plan to maintain an AppImage as Flatpak already solves the same problem with better sandboxing and automatic updates. Closing this, but feel free to reopen if you have specific issues with the Flatpak installation!

**Comment by shuvashish76** [comment]
Requested at https://github.com/pkgforge-dev/Anylinux-AppImages/issues/527
If anyone interested please test. Unofficial AppImages available at https://github.com/pkgforge-dev/Planify-AppImage/releases
